WooCommerce/Classes/POS/ViewModels/ItemListViewModel.swift

```diff
@@ -1,23 +1,19 @@
 import Combine
 import SwiftUI
 import protocol Yosemite.POSItem
-import protocol Yosemite.POSItemProvider
 
 final class ItemListViewModel: ItemListViewModelProtocol {
+    let posModel: PointOfSaleAggregateModelProtocol
```

Comment:
Not sure if it's worth it since we're removing this class soon, but we can make this const private:
```suggestion
    private let posModel: PointOfSaleAggregateModelProtocol
```

Reply:
Good spot but I'll just leave it :).